node/forest_libp2p/src/service.rs

```diff
@@ -158,6 +158,7 @@ pub struct Libp2pService<DB, P: StoreParams> {
     network_receiver_out: flume::Receiver<NetworkEvent>,
     network_sender_out: flume::Sender<NetworkEvent>,
     network_name: String,
+    genesis_hash: Cid,
```

Comment:
maybe let's use `genesis_cid`, `Cid` is not precisely a hash (it does contain it, though)

Reply:
Fixed